client/app/components/NavBar/index.js

```diff
@@ -3,9 +3,6 @@ import PropTypes from 'prop-types';
 import { Link } from 'react-router';
 import cn from 'classnames';
 
-// Styles
-import logo from '/Users/leseanjohnson/Documents/idicia/assets/images/svgs/IDICIA-sample.svg';
```

Comment:
Please consider why this wouldn't work on any machine other than your own :thinking: